.concourse/pipeline.yaml.gomplate

```diff
@@ -430,7 +424,9 @@ jobs:
         - -ce
         - |
           cd kubecf-{{ $branch }}
-          ./dev/build.sh ../output
+          VERSION="$(./scripts/version.sh)"
+          TARGET_FILE=../output/kubecf-${VERSION}.tgz make kubecf-build
```

Comment:
If we're overriding the target file name anyway, can we not embed the timestamp in the initial file name? Why do we need the `mv` afterwards?

Reply:
True, simplified.